### ISSUE TITLE
Decide how to handle meta info

### ISSUE THREAD

**Issue by mwickett** [sagecomm/reporter#8]
We'll probably need to do some kind of "front matter" type thing to handle content that goes on the cover page:

* report title
* client name
* date
* ?

**Comment by mwickett** [comment]
We could include these as variables in `app.js` and pass them through to the markup. Or create the equivalent of a `.env` file if we don't want to mix them into `app.js`. That might be better.

**Comment by zchsh** [comment]
Or could we integrate these right into the Paper doc? eg

**Comment by mwickett** [comment]
Good call. Could maybe use a code block or something so that there's a really clear visual differentiation in the document.

**Comment by mwickett** [comment]
The only consideration there is that we'll need to rely on Javascript to move all that stuff up out of the markdown element. Not hard to do, but somewhat brittle perhaps? Or, if we want to get really fancy, we create a pipeline that grabs the MD from paper, extracts stuff that we tell it to, and then passes it along to the `.sgr` template file, and makes the extracted content available as normal local variables? Might be overcomplicating it.
